examples/.experimental/next-app-dir/src/trpc/package.json

```diff
@@ -1,10 +1,10 @@
 {
   "name": "trpc-api",
-  "typings": "server.ts",
+  "typings": "server-fetcher.ts",
   "exports": {
     ".": {
-      "types": "./server.ts",
-      "react-server": "./server.ts",
+      "types": "./server-fetcher.ts",
```

Comment:
`-fetcher?` 🤔

Reply:
Let's keep it in one example, you can see how I started splitting it up a bit more on my branch

Reply:
You were doing actions though?